src/wizards/clusters.ts

```diff
@@ -196,7 +197,7 @@ function createEditClusterForm(cluster: Cluster | undefined, clusterSettings: Cl
                     cluster = createCluster(data);
                 }
                 // Save cluster
-                saveCluster(true, data, cluster, clusterSettings, clientAccessor, explorer, selectCluster);
+                await saveCluster(data, cluster);

```

Comment:
You have managed select cluster in SaveCommandHandler, it's nice, but you should remove all method which use `selectCluster` parameter;

Reply:
done

Reply:
You missed some code.

 * in commons.ts => addClusterCommandHandler.execute(); (line 48)
 * https://github.com/jlandersen/vscode-kafka/pull/189#discussion_r676464191